index.js

```diff
@@ -45,6 +39,15 @@ module.exports = async function hubdown (markdownString, opts = {}) {
     content = parsed.content
   }
 
+  const renderer = remark()
+    .use(opts.runBefore)
+    .use(slug)
+    .use(autolinkHeadings, { behavior: 'wrap' })
+    .use(inlineLinks)
+    .use(emoji)
+    .use(highlight)
+    .use(html, { sanitize: false })
```

Comment:
```suggestion
    .use([hljs, html], { sanitize: false })
```

Reply:
looks like `hljs` got lost in the shuffle

Reply:
Hmm no, i just renamed it to highlight (in the other PR).

Reply:
got it.